sample_logs/graphdb/create_graph.cypher

```diff
@@ -0,0 +1,98 @@
+CREATE (p1:Person {name: 'John Doe', birth_date: '1980-01-01'})
```

Comment:
I feel like this file should live in the `/data` directory. Happy to leave it here if we don't feel it should be there

Reply:
I agree, I'll move the file to /data